ssg/build_cpe.py

```diff
@@ -197,6 +189,21 @@ def to_xml_element(self, cpe_oval_filename):
         cpe_item_check.text = self.check_id
         return cpe_item
 
+    @classmethod
+    def from_yaml(cls, yaml_file, env_yaml=None, product_cpes=None):
+        cpe_item = super(CPEItem, cls).from_yaml(yaml_file, env_yaml, product_cpes)
+        # the is_product_cpe attribute is stored as string
+        # convert it to boolean
+        if cpe_item.is_product_cpe:
+            if cpe_item.is_product_cpe.lower() == "true":
```

Comment:
It almost looks like that a helper function `is_yaml_true` operating in this case on `cpe_item.is_product_cpe` would come in handy here and on other places as well - could also help to reduce need for existing comments.

Reply:
Fixed in commit 9a38533851ae5004d89f5c424a1736f1de8bff5f